### PR TITLE
test: Implemented remaning workspace-type apis

### PR DESCRIPTION
**Pull request by ealemayehu**
Implemented remaining workspace-type apis
    
1. Delete for configurations
2. Get for workspace-type-candidates
3. Get for config-vars

### CODE REVIEW

File: main/integration-tests/support/resources.js [hunk 1]
```diff
@@ -19,8 +19,9 @@ const Projects = require('./resources/projects/projects');
 const Indexes = require('./resources/indexes/indexes');
 const CurrentUser = require('./resources/current-user');
 const PublicAuthProviderConfigs = require('./resources/public-auth-provider/public-auth-provider-configs');
-const WorkspaceTypes = require('./resources/workspace-types/workspace-types.js');
-const AwsAccounts = require('./resources/aws-accounts/aws-accounts.js');
+const WorkspaceTypes = require('./resources/workspace-types/workspace-types');
+const WorkspaceTypeCandidates = require('./resources/workspace-type-candidates/workspace-type-candidates.js');
```
Comment (hatimkhan) [inline main/integration-tests/support/resources.js]: Can you remove '.js' from require

File: main/integration-tests/support/resources/workspace-type-candidates/workspace-type-candidates.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const CollectionResource = require('../base/collection-resource');
+
+class WorkspaceTypeCandidate extends CollectionResource {
```
Comment (hatimkhan) [inline main/integration-tests/support/resources/workspace-type-candidates/workspace-type-candidates.js]: I think the class name is missing 's'.

File: main/integration-tests/support/resources.js (hunk 1 above)
Comment (ealemayehu) [reply main/integration-tests/support/resources.js]: Fixed. Thanks!

File: main/integration-tests/support/resources/workspace-type-candidates/workspace-type-candidates.js (hunk 2 above)
Comment (ealemayehu) [reply main/integration-tests/support/resources/workspace-type-candidates/workspace-type-candidates.js]: Fixed. Thanks!

**Review by hatimkhan — DISMISSED** [review]

**Review by SanketD92 — APPROVED** [review]

**Pull request merged**